kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtComponentServiceImpl.java

```diff
@@ -59,6 +62,8 @@
     private static final String SERVICE_FACTORY_PID = "service.factoryPid";
     private static final String KURA_UI_SERVICE_HIDE = "kura.ui.service.hide";
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
Then also consider this: either have one constant for all instances of the class that gets initialized once or have so many instances as there are instances of running method that get initialized every time the method is ran. And considering the time we wasted on this pointless debate, we already made up for all the initialization of constants in this and all other classes Kura will ever use.